src/components/popover/popover.jsx

```diff
@@ -86,14 +104,18 @@ const Popover = forwardRef(function Popover(props, ref) {
 });
 
 Popover.propTypes = {
+  autoflush: PropTypes.bool,
   children: PropTypes.node,
+  flush: PropTypes.oneOf(['left', 'right']),
```

Comment:
For these kinds of props, we can to be exhaustive in our test suite. We need one for setting it to `left` and another for `right`. 

We skip approximately large prop values like `string` or `number` in favor of a single or few good cases.

Reply:
Yep, good call! - Good call! e631eac6